### DISCUSSION TITLE
Destination - > Home missing(?)

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#15214] [Q&A]
When I select a destination for navigation there was (in former OSMAnd-Versions) "home" selectable in the menue. With the new version this is missing (in favorites my "home" - adress is still visible) 
![Screenshot_20220902-194506](https://user-images.githubusercontent.com/62500258/188209591-e43372dd-699f-468c-94da-a808947f4893.png)

Bug? Feature?

**Comment by ThomasMeister123** [comment]
Screenshot of old version
![Screenshot_2022-09-02-19-44-10](https://user-images.githubusercontent.com/62500258/188210985-ef5c1e23-dd3d-44e4-80a1-be5880acb54c.png)
"Zuhause" (="Home"; I entered my adress) and  "Arbeit" (="Work"; I dont entered any adress here)

**Comment by vshcherb** [comment]
It's not a bug, it's a different screen: 2n you didn't specify target, 1st you've already built a route to a specified target
